helmfile.d/charts/hnc/controller/values.yaml

```diff
@@ -105,3 +105,5 @@ certificate:
 serviceMonitor:
   create: true
   relabelings: []
+
+webhookMatchConditions: false
```

Comment:
I think it makes sense to put a comment there saying it will be changed to true by default closer to EOL of v1.27.
But that comment should be added both here and in the config folder.